synapse/tests/test_lib_stormlib_gen.py

```diff
@@ -344,3 +344,31 @@ async def test_stormlib_gen_cryptoX509Cert(self):
             nodes = await core.nodes('yield $lib.gen.cryptoX509CertBySha256($sha256)', opts=opts)
             self.len(1, nodes)
             self.eq(nodes[0].repr(), crypto)
+
+    async def test_stormlib_gen_service(self):
+
+        async with self.getTestCore() as core:
+
```

Comment:
need additional tests to confirm multiple calls return the same node and confirm secondary property deconfliction with manually created nodes